docs/server.md

```diff
@@ -79,133 +78,73 @@ After this deep dive, you might also be interested in:
 
 ## Model Files
 
-The rest of this topic focus on serving of model files without much discussion on the model files themselves, where they come from, and how they're made. Long story short: it's a zip archive with the parameters, weights, and metadata that define a model that has been trained already. If you want to know more about the model files, take a look at the [export tool documentation](export.md).
+The rest of this topic focus on serving of model files without much discussion on the model files themselves, where they come from, and how they're made. Long story short: it's a zip archive with the parameters, weights, and metadata that define a model that has been trained already. If you want to know more about the model files, take a look at the [model-archiver documentation](https://github.com/awslabs/mxnet-model-server/blob/master/model-archiver/README.md)).
 
 ## Command Line Interface
 
 ```bash
 $ mxnet-model-server --help
-usage: mxnet-model-server [-h] --models KEY1=VAL1 KEY2=VAL2...
-                          [KEY1=VAL1 KEY2=VAL2... ...] [--service SERVICE]
-                          [--gen-api GEN_API] [--port PORT] [--host HOST]
-                          [--gpu GPU] [--log-file LOG_FILE]
-                          [--log-rotation-time LOG_ROTATION_TIME]
-                          [--log-level LOG_LEVEL]
-                          [--metrics-write-to {log,csv,cloudwatch}]
+usage: mxnet-model-server [-h] (--start | --stop) [--mms-config MMS_CONFIG]
```

Comment:
start and stop are optional. You might need to change the brackets from () to []